Update cache-how-to-scale.md - Explain that the cache IP address may change (#79831)
Azure/azure-content: articles/azure-cache-for-redis/cache-how-to-scale.md

- When you scale a **Standard** cache to a different size or to a **Premium** cache, one of the replicas is shut down and reprovisioned to the new size and the data transferred over, and then the other replica does a failover before it's reprovisioned, similar to the process that occurs during a failure of one of the cache nodes.
- When you scale out a clustered cache, new shards are provisioned and added to the Redis server cluster. Data is then resharded across all shards.
- When you scale in a clustered cache, data is first resharded and then cluster size is reduced to required shards.

> - In some cases, such as scaling migrating your cache to a different cluster, the underlying IP address of the cache may change. This is done in a way that is transparent to most applications (by updating the DNS records for the cache), but if you use an IP address to configure the connection to your cache, or to configure NSGs or firewalls allowing traffic to the cache, your application may have trouble connecting from around the time that the DNS record updates.

Reviewer: ```suggestion
- In some cases, such as scaling or migrating your cache to a different cluster, the underlying IP address of the cache can change. The DNS records for the cache changes and is transparent to most applications. However, if you use an IP address to configure the connection to your cache, or to configure NSGs, or firewalls allowing traffic to the cache, your application might have trouble connecting sometime after that the DNS record updates.
```
I would love to look at this with automated tooling to check the Acrolinx score. Maybe we could do that with a bug after the PR?
